packages/client/components/EditingStatus/EditingStatusText.tsx

```diff
@@ -30,6 +31,12 @@ const useTimeFrom = (timestamp: string) => {
   return timeFrom
 }
 
+const HeaderName = styled('div')({
```

Comment:
+1 Let's make naming consistent with other elements, as there is no Header. For example just `<Name>`